.vuepress/components/Home.vue

```diff
@@ -0,0 +1,314 @@
+      transform: translateY(-2px);
+      transition-duration: 0.1s;
+    }
+
+    &:active {
+      transition-duration: 0s;
+      opacity: 0.7;
+    }
+
+    &__tag {
+      color: var(--tag-text-color);
+      font-size: 14px;
+      line-height: 20px;
+      letter-spacing: 0.03em;
+      text-transform: capitalize;
+      background: var(--tag-background-color);
+      border-radius: 4px;
+      padding: 2px 8px;
+      width: fit-content;
+    }
+
+    &__title {
+      font-weight: 500;
+      font-size: 1.25rem;
+      line-height: 1.75rem;
+      letter-spacing: -0.01em;
+      color: #000000;
+      margin-bottom: 0.5rem;
+    }
+
+    &__desc {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      letter-spacing: 0.03em;
+      color: rgba(0, 0, 0, 0.667);
+    }
+  }
+}
+
+.stack {
+  display: grid;
+  gap: 1.5rem;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  margin-bottom: 4rem;
+
+  &__item {
+    color: var(--color-text, black);
+    transition: box-shadow 0.25s ease-out, transform 0.25s ease-out, opacity 0.4s ease-out;
+
+    &:active {
+      opacity: 0.7;
+      transition-duration: 0s;
+    }
+
+    &__image {
+      width: 100%;
+      object-fit: cover;
+    }
+
+    &__h1 {
+      font-size: 1.25rem;
+      line-height: 1.5rem;
+      font-weight: 600;
+      margin-top: 12px;
+      margin-bottom: 12px;
+    }
+
+    &__p {
+      font-size: 0.875rem;
+      color: rgba(22, 25, 49, 0.65);
+      line-height: 1.25rem;
+    }
+
+    &__text {
+      text-align: left;
+    }
+  }
+}
+
+@media screen and (max-width: 1136px) {
+  .p {
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+  }
+}
+
+@media screen and (max-width: 832px) {
+  .h1 {
+    padding-top: 3.5rem;
+  }
+
+  .search__container {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 752px) {
+  .search {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .h1 {
+    font-size: 2rem;
+    line-height: 2.25rem;
+    margin-bottom: 1rem;
+  }
+
+  .h2 {
+    font-size: 1.5rem;
+    line-height: 2rem;
+    margin-top: 3rem;
+    margin-bottom: 0.75rem;
+  }
+
+  .p__alt {
+    font-size: 1rem;
+    line-height: 1.5rem;
+  }
+
+  .sections {
+    margin-bottom: 0;
+    margin-top: 2rem;
+    grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+    margin-left: -1rem;
+    margin-right: -1rem;
+
+    &__item {
+      margin-bottom: 0;
+
+      &__wrapper {
+        padding-bottom: 1.25rem;
+      }
+
+      &:last-child .sections__item__wrapper {
+        border-bottom: none;
+      }
+    }
+
+    &__wrapper {
+      position: relative;
+      padding: 0.1px 1rem 1rem;
+      background: white;
+      border-radius: 0.5rem;
+    }
+  }
+
+  .stack {
+    grid-template-columns: repeat(auto-fill, minmax(240px, 1fr));
+    margin-bottom: 3rem;
+
+    &__item {
+      &__text {
+        grid-template-columns: 3rem 1fr;
+        text-align: start;
+      }
+
+      &__h1 {
```

Comment:
All of this can be removed now that we're reducing the size of the the base styles for this element.